src/utils/Proof.js

```diff
@@ -62,13 +62,14 @@ class Proof {
   }
 
   /**
-   * Get proof data
+   * Get proof data, if blockNumber is not passed it will generate proof for
+   * latest block.
    *
    * @param {Web3} web3 web3 instance of chain from which proof is generated.
    * @param {string} index Storage index.
    * @param {string} address Address of ethereum account for which proof needs
    *                         to be generated.
-   * @param {String[]} keys Array of keys for a mapping in solidity.
+   * @param {string[]} keys Array of keys for a mapping in solidity.
    * @param {string} blockNumber Block number in hex.
```

Comment:
Optional parameters are wrapped in brackets:
```suggestion
    * @param {string} [blockNumber] Block number in hex.
```